packaging/packager_rpm.sh

```diff
@@ -20,7 +20,7 @@ dnf -y install tesseract tesseract-devel
 dnf -y install gcc-c++
 pip3 install pytesseract==0.3.4 tesserocr==2.5.1
 # deep learning
-pip3 install torch==1.8.1 torchvision==0.9.1
+pip3 install torch==2.0.0 torchvision==0.15.1
```

Comment:
I think this is going too far in the version and it might break our code but of course would be great if it works or needs just a few adjustments.